extensions/spark/kyuubi-extension-spark-common/src/main/scala/org/apache/kyuubi/sql/KyuubiSQLConf.scala

```diff
@@ -190,4 +190,43 @@ object KyuubiSQLConf {
       .version("1.7.0")
       .booleanConf
       .createWithDefault(true)
+
+  val FINAL_WRITE_STAGE_RESOURCE_ISOLATION_ENABLE =
```

Comment:
Nit:

Is it possible to standardize the style of variables, such as the end of _ENABLE`D` or simply remove _ENABLE?

e.g:

`FINAL_STAGE_CONFIG_ISOLATION_WRITE_ONLY` or `ZORDER_USING_ORIGINAL_ORDERING_ENABLED `

Reply:
thank you for the nit, it should be `enabled`